### ISSUE TITLE
Separator line in tray menu between engine status and the other items

### ISSUE THREAD

**Issue by bertfrees** [daisy/pipeline-ui#131]
The engine status is always gray. It would make it stand out as a special item even more if there would be a separator line beneath it.

**Comment by rdeltour** [comment]
Yeah, I agree that the UX could be improved here. the current solution uses a "disabled" menu item, which is (by design) very low contrast (at least on macOS), and not very readable.

Another possibility would be to integrate the engine state (running/stopped) to the tray icon (e.g. use an icon with an oblique stroke when the engine is not running…) and move the port number detail to the about box.

**Comment by marisademeglio** [comment]
The engine state and address are available in the about box. 

Worth noting: the "engine state" menu item in the tray turns into "start the engine" if the engine stops while the application is still open (though the engine stopping suddenly is not common behavior so you may have not seen this yet).

If we remove the engine state menu item, then we would (?) be removing this function too.

**Comment by bertfrees** [comment]
> Another possibility would be to integrate the engine state (running/stopped) to the tray icon

Yes, thought about this too.

> and move the port number detail to the about box.

Indeed I would remove the "on port xxx". I wouldn't remove the whole menu item.

**Comment by marisademeglio** [comment]
This is what I can do now:

<img width="138" alt="Screenshot 2023-03-16 at 16 39 49" src="https://user-images.githubusercontent.com/734730/225775802-d2d0893e-d583-4f40-9956-27198abf658c.png">

Alt: tray menu says "engine status, separator, open window, new job, settings, about, separator, quit"

? 

And then we can make a new issue for using the icon to show status.

**Comment by bertfrees** [comment]
Perfect
